parser/openfoodfacts_taxonomy_parser/unparser.py

```diff
@@ -118,14 +127,16 @@ def rewrite_file(self, filename, lines):
             for line in lines:
                 file.write(line + "\n")
 
-    def __call__(self, filename):
-        lines = self.iter_lines()
+    def __call__(self, filename, branch_name):
+        filename = self.normalized_filename(filename)
+        branch_name = self.normalizing(branch_name, char="_")
+        multi_label = self.create_multi_label(filename, branch_name)
+        lines = self.iter_lines(multi_label)
         self.rewrite_file(filename, lines)
 
 
 if __name__ == "__main__":
-    import sys
-
     filename = sys.argv[1] if len(sys.argv) > 1 else "test"
+    branch_name = sys.argv[2] if len(sys.argv) > 1 else "branch"
     write = WriteTaxonomy()
-    write(filename)
+    write(filename, branch_name)
```

Comment:
as for Parser, make taxonomy name a distinct parameter, not necessarily correlated to filename.